keyboards/takashicompany/mirageix/keymaps/default/keymap.c

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    LAYOUT(
```

Comment:
Could you add explicit indices to make this a bit more readable:
```c
[0] = LAYOUT(),
[1] = LAYOUT(),
[3] = LAYOUT(),
// ...
```

Reply:
Thanks.

I fixed!